src/main/java/hardware/Swerve.java

```diff
@@ -124,6 +125,10 @@ public void periodic() {
   public void logPositions() {
     SwerveLogBuilder logs = new SwerveLogBuilder();
     DriverUI.field.setRobotPose(getPose());
+    
+    logs.addRealModuleStates(new double[]{ 0, 0, 0, 0, 0, 0, 0, 0 });
+    logs.addRobotRotation(getPose().getRotation().getDegrees());
```

Comment:
This should be in the constructor of Swerve.java, not in logPositions. at the end of the constructor it should look something like
```java
    logs.addRealModuleStates(new double[]{ 0, 0, 0, 0, 0, 0, 0, 0 });
    logs.addRobotRotation(getPose().getRotation().getDegrees());
    logs.sendValues();
```
So that there are default values assigned in the network tables when the robot initially boots up.

Reply:
Yes, thank you, I forgot last night that log positions get run periodically. I'll fix that up today and merge